packages/core/src/views/account-center/WalletRow.svelte

```diff
@@ -56,6 +56,14 @@
       }
     }
   }
+
+  async function copyWalletAddress(address: string) {
```

Comment:
Nice! It might be useful to pull this into utils, I could see us using it elsewhere.
Is there any user feedback when copying occurs?